client/src/session_retry.rs

```diff
@@ -108,8 +109,8 @@ impl SessionRetryPolicy {
         Self::new(session_timeout, 0, 0)
     }
 
-    fn last_attempt_default() -> DateTime<Utc> {
-        Utc.ymd(1900, 1, 1).and_hms(0, 0, 0)
+    fn last_attempt_default() -> DateTime {
+        DateTime::ymd(1900, 1, 1)
```

Comment:
Just make sure this also sets clock to 00:00:00

Reply:
Yes, I double checked and this is indeed the case. See how the time is actually set [here](https://github.com/svanharmelen/opcua/blob/feature/time-keeping/types/src/date_time.rs#L247).